ci-operator/step-registry/openshift-qe/rosa-loaded-upgrade/OWNERS

```diff
@@ -0,0 +1,21 @@
+approvers:
```

Comment:
Lets update this file to be a symbolic link to the OWNERS file in the openshift-qe folder just like this one: https://github.com/openshift/release/blob/6b22ed88890704e4a4b2cb50d92abf1e4a4dffa8/ci-operator/step-registry/openshift-qe/rosa-upgrade-postcheck/OWNERS

Reply:
fixed in new code